### DISCUSSION TITLE
Tab activity color

### DISCUSSION THREAD

**Discussion by aryklein** [kovidgoyal/kitty#3858] [Q&A]
Is there any way to change the tab color or the tab title color when you have activity, instead of adding a symbol (tab_activity_symbol)? If not, is there a way to add a a blank space in the `tab_activity_symbol` to keep away the symbol from the tab tittle with a space?

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/commit/e48adf7a07d039920784c42e97729001c10e5227

**Reply by aryklein** [reply]
Thanks
